src/chat/UnreadNotice.js

```diff
@@ -30,6 +30,11 @@ const styles = createStyleSheet({
     color: 'white',
     paddingRight: 4,
   },
+  unreadNumberExceeds: {
```

Comment:
```
Issue #4904 Solved
```

Please follow our [commit message style](https://github.com/zulip/zulip-mobile/blob/master/docs/style.md#git-commits-commit-messages).

Reply:
Thankyou for pointing this out. I have made the edit.

Reply:
Ah, I see you've changed the title, but I was talking about your first comment in the PR, [this one](https://github.com/zulip/zulip-mobile/pull/4979#issue-722716437). I think GitHub also calls this the "description" of the PR.